Gemfile

```diff
@@ -85,6 +85,7 @@ gem "sneakers", "~> 2.11" # rabbitMQ background processing
 gem "state_machines-activerecord"
 gem "strip_attributes"
 gem "turbo-rails", "~> 1.0"
-gem "view_component", "~> 2.56.2" # https://github.com/github/view_component/issues/1390
+gem "view_component", "~> 3.5"
```

Comment:
Surprise! We're using an old version of View Components, which causes all sorts of problems for form rendering.